### PR TITLE
Consume dedicated org member suggestion endpoint

### PR DESCRIPTION
**Pull request by cvializ**
@adamjgrant

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileInviteSearch.js [hunk 1]
```diff
@@ -179,10 +170,7 @@ angular.module('kifi')
           }
 
           // Do the magic to get the contacts given the organization id.
-          orgProfileService
-            .getOrgLibraries(scope.organization.id)
-            .then(getFirstLibrary)
-            .then(getSearchContacts)
+          orgProfileService.suggestOrgMember(scope.organization.id, opt_query)
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileInviteSearch.js:173]: Nice

**Comment by adamjgrant** [comment]
What is this for (in asana)?

**Comment by cvializ** [comment]
https://app.asana.com/0/inbox/39814736029950/44137262387823/44566760584143
It wasn't assigned to me, but I had to do some work for it.

**Comment by adamjgrant** [comment]
:+1:

**Pull request merged**